src/coreclr/src/nativeaot/Runtime.Base/src/System/Runtime/DispatchResolve.cs

```diff
@@ -19,7 +19,7 @@ internal static unsafe class DispatchResolve
             // Use the dynamic module resolver if it's present
             if ((dynamicModule != null) && (dynamicModule->DynamicTypeSlotDispatchResolve != null))
```

Comment:
Maybe we can also cache `DynamicTypeSlotDispatchResolve` in a local to avoid calling the property twice?